contrib/win32/openssh/OpenSSHUtils.psm1

```diff
@@ -829,4 +829,25 @@ function Enable-Privilege {
     $type[0]::EnablePrivilege($Privilege, $Disable)
 }
 
-Export-ModuleMember -Function Repair-FilePermission, Repair-SshdConfigPermission, Repair-SshdHostKeyPermission, Repair-AuthorizedKeyPermission, Repair-UserKeyPermission, Repair-UserSshConfigPermission, Enable-Privilege, Get-UserAccount, Get-UserSID, Repair-AdministratorsAuthorizedKeysPermission, Repair-ModuliFilePermission, Repair-SSHFolderPermission, Repair-SSHFolderFilePermission, Repair-SSHFolderPrivateKeyPermission
+Function Add-Path {
```

Comment:
This adds a path to the Machine scope 'PATH' variable only, so the function name should reflect that.  Or we could add another 'Scope' parameter to determine which environment scope should be affected ('Machine', 'Process', 'User'), and make the function more general.